views/random-beer.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each randomBeer}}
+<div class="beers">
+   {{> beerpartial this}}
+    <p>
+        {{food_pairing}}
+    </p>
+    <p>
+        {{brewers_tips}}
+    </p>
+</div>
+{{/each}}
```

Comment:
A tiny thing: It would be nice to have the food pairing and tips introduced: 

<p> Food pairing: {{food_pairing  </p> 

Right now I don't know there's just a bunch of text and it's confusing.